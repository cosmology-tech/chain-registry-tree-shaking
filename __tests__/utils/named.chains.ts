import { osmosis } from 'interchain-registry/named';

it('list', () => {
    const result = osmosis.assets.assets.map(asset => asset.name);
    expect(result).toMatchSnapshot();
})

it('chain', () => {
    const result = osmosis.chain
    expect(result.chain_id).toMatchSnapshot();
})