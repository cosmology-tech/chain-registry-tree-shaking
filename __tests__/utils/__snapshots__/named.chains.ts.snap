// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chain 1`] = `"osmosis-1"`;

exports[`list 1`] = `
[
  "Osmosis",
  "Ion DAO",
  "USD Coin (Axelar)",
  "Ether",
  "Wrapped Bitcoin (Axelar)",
  "Tether USD (Axelar)",
  "Dai Stablecoin",
  "Binance USD",
  "Cosmos Hub",
  "Cronos POS Chain",
  "Binance Coin",
  "Polygon",
  "Avalanche",
  "Luna Classic",
  "Juno",
  "Wrapped Polkadot (Axelar)",
  "Evmos",
  "Kava",
  "Secret Network",
  "TerraClassicUSD",
  "Stargaze",
  "Chihuahua",
  "Persistence",
  "pSTAKE Finance",
  "Akash",
  "Regen",
  "Sentinel",
  "IRISnet",
  "Starname",
  "e-Money",
  "e-Money EUR",
  "LikeCoin",
  "ixo",
  "BitCanna",
  "BitSong",
  "Ki",
  "Medibloc",
  "bostrom",
  "Comdex",
  "Cheqd",
  "Lum Network",
  "Vidulum",
  "Desmos",
  "Dig Chain",
  "Sommelier",
  "Band Protocol",
  "Konstellation",
  "Umee",
  "Gravity Bridge",
  "Decentr",
  "Marble",
  "Carbon",
  "Cerberus",
  "Fetch.ai",
  "AssetMantle",
  "Neta",
  "Injective",
  "TerraClassicKRW",
  "Microtick",
  "Sifchain",
  "Shentu",
  "Hope Galaxy",
  "Racoon",
  "Frax",
  "Wrapped Bitcoin (Gravity Bridge)",
  "Ether (Gravity Bridge)",
  "USD Coin (Gravity Bridge)",
  "DAI Stablecoin (Gravity Bridge)",
  "Tether USD (Gravity Bridge)",
  "Block",
  "Provenance",
  "Galaxy",
  "DHK",
  "JunoSwap",
  "MEME",
  "Another.Software Validator Token",
  "JoeDAO",
  "Luna",
  "Rizon",
  "Kava Hard",
  "Kava Swap",
  "Chainlink",
  "GenesisL1",
  "Aave",
  "ApeCoin",
  "Maker",
  "Rai Reflex Index",
  "Shiba Inu",
  "Kujira",
  "Tgrade",
  "Echelon",
  "Odin Protocol",
  "GEO",
  "O9W",
  "LVN",
  "Moonbeam",
  "Gelotto",
  "GKey",
  "Crescent",
  "LumenX",
  "Oraichain",
  "Cudos",
  "Kava USDX",
  "Agoric",
  "Inter Stable Token",
  "StakeEasy seJUNO",
  "StakeEasy bJUNO",
  "Stride",
  "Stride Staked ATOM",
  "Stride Staked STARS",
  "Solarbank DAO",
  "StakeEasy SEASY",
  "Axelar",
  "Rebus",
  "Teritori",
  "Stride Staked JUNO",
  "Stride Staked OSMO",
  "MuseDAO",
  "Lambda",
  "USK",
  "Unification",
  "Jackal",
  "Alter",
  "Button",
  "Shade (old)",
  "SIENNA",
  "SCRT Staking Derivatives",
  "BeeZee",
  "Fanfury",
  "Acrechain",
  "CMST",
  "Imversed",
  "Medas Digital Network",
  "POSTHUMAN",
  "Amber",
  "Onomy",
  "PSTAKE staked ATOM",
  "Dyson Protocol",
  "Hopers",
  "Arable USD",
  "Planq",
  "Fantom",
  "Canto",
  "Quicksilver Liquid Staked STARS",
  "Wynd DAO Governance Token",
  "USD Coin (Polygon)",
  "USD Coin (Avalanche)",
  "Mars Hub",
  "Ciento Token",
  "Stride Staked LUNA",
  "Stride Staked EVMOS",
  "nRide Token",
  "8ball",
  "Quicksilver Liquid Staked ATOM",
  "Harbor",
  "Quicksilver Liquid Staked Regen",
  "Juno Fox",
  "Quicksilver",
  "Arkhadian",
  "Quicksilver Liquid Staked OSMO",
  "Frienzies",
  "Migaloo",
  "Guardian",
  "Mini Punks",
  "ShibaCosmos",
  "Sikoba Token",
  "Nature Carbon Ton",
  "Celestims",
  "Osmosis Doge",
  "Apemos",
  "Invaders",
  "Doge Apr",
  "Catmos",
  "Summit",
  "OmniFlix",
  "Spacer",
  "LIGHT",
  "Silk",
  "Mille",
  "Manna",
  "Filecoin",
  "Void",
  "Shade",
  "Bluzelle",
  "Arbitrum",
  "Silica",
  "Pepec",
  "Pepe",
  "IBC Index",
  "Coinbase Wrapped Staked ETH",
  "Rocket Pool Ether",
  "Staked Frax Ether",
  "Wrapped Lido Staked Ether (Axelar)",
  "Gitopia",
  "Lion DAO",
  "Stride Staked UMEE",
  "Staked IBCX",
  "Nolus",
  "Lion Cub DAO",
  "BLUE CUB DAO",
  "Neutron",
  "Casa",
  "Composable",
  "Kusama",
  "Polkadot",
  "Quasar",
  "Archway",
  "EmpowerChain",
  "WATR",
  "KYVE",
  "Tether USD",
  "ERIS Amplified OSMO",
  "Sei",
  "Quicksilver Liquid Staked SOMM",
  "Passage",
  "Stride Staked SOMM",
  "Solana",
  "Bonk",
  "Tether USD (Wormhole)",
  "Sui",
  "Aptos Coin",
  "MantaDAO",
  "Licorice",
  "USD Coin (Wormhole)",
  "Wrapped Ether (Wormhole)",
  "USD Coin",
  "Real Yield ETH",
  "XPLA",
  "OIN STORE OF VALUE",
  "Neokingdom DAO",
  "Realio Network",
  "CDT Stablecoin",
  "Membrane",
  "SGE",
  "StaFi Hub",
  "rATOM",
  "Stardust STRDST",
  "Dora Vota",
  "Coreum",
  "Celestia",
  "dYdX Protocol",
  "f(x)Core",
  "Nomic Bitcoin",
  "Nois",
  "OSMO Squared",
  "Unstake Fi",
  "Branch",
  "Wrapped Lido Staked Ether",
  "ATOM Squared",
  "BTC Squared",
  "Qwoyn",
  "Bostrom Hydrogen",
  "Bostrom Tocyb",
  "Bostrom Volt",
  "Bostrom Ampere",
  "Source",
  "Pyth Network",
  "PSTAKE staked OSMO",
  "Levana",
  "Puppy",
  "Newt",
  "milkTIA",
  "ASH",
  "RAC",
  "GUPPY",
  "Haqq Network",
  "Autism",
  "Page",
  "PURSE Token (Function X)",
  "Dog wif nunchucks",
  "Kleomedes",
  "Nym",
  "NYM",
  "Chihuahuawifhat",
  "AtomEconomicZone69JaeKwonInu",
  "Jun√∏ Apes",
  "WOOF",
  "Sneaky Productions",
  "Wrapped Bitcoin",
  "Badcoin",
  "Signal",
  "WOSMO",
  "TIA Squared",
  "Apollo DAO",
  "Stride Staked DYDX",
  "Stride Staked TIA",
  "Gelotto (Injective)",
  "Dymension Hub",
  "RAPTR",
  "Wormhole USDC(Avalanche)",
  "Sail",
  "Nomic",
  "BADKID",
  "Yieldmos Coin",
  "TORO",
  "SHARK",
  "BackBone Labs Liquid Staked OSMO",
  "Branez",
  "CVN",
  "LAB",
  "IBC",
  "Tinkernet",
]
`;
